app/components/course-page/course-stage-step/community-solution-card/feedback-section.ts

```
@@ -0,0 +1,87 @@
+    if (toggleUpvote) {
+      this.flashSuccessMessageTask.perform();
+      await this.args.solution.upvote(this.args.metadataForUpvote || {});
+    } else if (toggleDownvote) {
+      this.flashSuccessMessageTask.perform();
+      await this.args.solution.downvote(this.args.metadataForDownvote || {});
+    } else if (toggleUnvote) {
+      await this.args.solution.unvote({});
+    }
+  });
+
+  @action
```

Biome (1.9.4)

[error] This parameter is used before its declaration.

The parameter is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)